content/_index/features/hosted.md

```diff
@@ -1,8 +1,10 @@
 +++
 title = "Hosted Service"
 weight = 20
-icon = "fas fa-cogs"
+
+[asset]
+  icon = "fas fa-cogs"
 +++
 
-Let us worry about the global and fault tolerant backend infrastructure while you control and manage your redirect data within your DNS records.  
+Let us worry about the global and fault tolerant backend infrastructure while you control and manage your redirect data within your DNS records.
```

Comment:
You definitely don't like intentional line breaks in markdown ;)

Reply:
My vscode is configured to trim end of the line whitespace by default, I disable it per project but forgot for this project.